Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM bitnami/python:3.7-prod
+
+ENV PIP_NO_CACHE_DIR=false \
+    PIPENV_HIDE_EMOJIS=1 \
+    PIPENV_IGNORE_VIRTUALENVS=1 \
+    PIPENV_NOSPIN=1
+
+WORKDIR /bot
+COPY . .
+
+# Update setuptools by removing egg first, add other dependencies
+RUN rm -r /opt/bitnami/python/lib/python3.*/site-packages/setuptools* && \
```

Comment:
This can be moved before `COPY` to improve layer caching.

Reply:
The setuptools install is no longer required with the up-to-date official python image, but the pipenv install was moved to before the `COPY` now, thanks.